hledger/Hledger/Cli/BalanceView.hs

```diff
@@ -129,7 +129,7 @@ balanceviewReport BalanceView{..} CliOpts{command_=cmd, reportopts_=ropts, rawop
         unless (no_total_ ropts' || cmd=="cashflow") . mapM_ putStrLn $ -- TODO temp
           [ "Total:"
           , "--------------------"
-          , padleft 20 $ showMixedAmountWithoutPrice (getSum amt)
+          , padLeftWide 20 $ showamt (getSum amt)
```

Comment:
Why switch to padLeftWide... for better layout of a multi-line amount ? Do you have an example by chance ? If it's noticeable, we should mention it in commit history, release notes etc.